src/core/qgstextrenderer.cpp

```diff
@@ -861,6 +869,12 @@ void QgsTextBackgroundSettings::readXml( const QDomElement &elem, const QgsReadW
     setPaintEffect( QgsApplication::paintEffectRegistry()->createEffect( effectElem ) );
   else
     setPaintEffect( nullptr );
+
+  const QDomElement symbolElem = backgroundElem.firstChildElement( QStringLiteral( "symbol" ) );
```

Comment:
Is "symbol" overly generic a string? No string feeling, just asking :)

Reply:
It's baked into how symbols are saved -- the alternative would be to embed another depth of element, which I'd rather avoid